_posts/2021-10-07-open-source-economics-is-not-what-you-think.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Open Source Economics (is not what you think)
+---
+
+"Open Source Economics" and the "Open Source Economy" are regularly discussed in the context of how to improve open source software's sustainability, contributor diversity and ecosystem quality. Too often, though, the use of the word "economics" brings incorrect assumptions about the problems to be solved.
+
+### 👎 What aren't "Open Source Economics"?
+
+When most people hear the term "economics" they tend to think about how 💵💶💷💴 flows around an economy and, particularly in a capitalist economy, how the allocation of capital affects the throughput of businesses operating in a free market.
+
+As a result, when people hear the term "open source economics" they tend to jump to the same conclusions: it's about how 💵💶💷💴 flows between and is invested in open source projects. This is not a bad conclusion, it results in the creation of tools such as [GitHub Sponsors](https://github.com/sponsors) that benefit many open source maintainers but it's not the whole picture.
+
+![GitHub Sponsors]({{ '/images/a/github_sponsors.png' | absolute_url }})
+
+When you look just at the financial side of open source projects you tend to assume any problems they suffer are due to a lack of investment and can be solved by adding more 💵💶💷💴. For example, if a project struggles to make releases, review and merge pull requests, close issues or answer discussions: if they had more money they could pay someone to do that work and more of it will get done.
+
+Unfortunately, money alone will not always fix any of these problems. Money can help but it needs correct investment and understanding of what the _real_ problem is.
+
+### 👍 What are "Open Source Economics"?
+
+An "economic problem" is one which requires the allocation of limited resources in order to solve a problem. In modern, capitalist economies: the limited resources are usually 💵💶💷💴. Where there is a labour constraint, this is solved by paying enough money to compensate or attract the right candidates to solve the problem.
+
+Open source software is a little different because of the high degree of specificity, limited transferability/substitutability and non-financial incentives of "labour" (maintainers). Many widely used open source projects are built by a single maintainer and, if so, this maintainer is the only person who can make all releases, review and merge all pull requests. Even with multiple maintainers, a widely used project will have a much smaller number of people doing this work than consuming the results. This means that domain knowledge and access control is limited to this one person or limited group of people.
```

Comment:
How is this different from the broader economy? Software engineers (and, for that matter, civil engineers, steel machinists, and surgeons) often have very specific skills that may command a premium in some workplaces or projects but not in others, and non-monetary factors matter too (pride, prestige, location, ethics, etc).

Reply:
Good points; making this more specific: 

> Open source software is a little different because of small pool of "labour" (maintainers) with knowledge of a project (perhaps single person) most of whom do their work voluntarily for no direct financial compensation.